includes/API/API.php

```
@@ -1230,74 +1627,116 @@ private function call_openai_api( $endpoint, $api_key, $model, $prompt, $system_
+     * @param string $system_prompt System prompt.
+     * @param int    $max_tokens    Max tokens.
+     * @param float  $temperature   Temperature.
+     * @param array  $images        Optional array of images for vision.
+     * @param int    $timeout       Request timeout in seconds.
+     *
+     * @return array Response with content and usage.
+     * @throws \Exception When API call fails.
      */
-    private function call_anthropic_api( $endpoint, $api_key, $model, $prompt, $system_prompt, $max_tokens, $temperature ) {
-        // Validate API key
+    private function call_anthropic_api( $endpoint, $api_key, $model, $prompt, $system_prompt, $max_tokens, $temperature, $images = [], $timeout = 60 ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$temperature'. (undefined)

(UnusedFormalParameter)